### DISCUSSION TITLE
Where I put the --upgrade flag ?

### DISCUSSION THREAD

**Discussion by ShmuelRonen** [vladmandic/sdnext#949] [Q&A]
### Issue Description

I use WSL 2.0 on windows 10 pro

**Comment by brknsoul** [comment]
`./webui.sh --upgrade`

Not sure why you're running in WSL, since there's no speed increase to be found there.
Either use CUDA (nvidia) or DirectML (amd) in Windows native, or boot up Linux and use CUDA or ROCm (amd) there.

In the future, ask questions in the Discussion tab.

**Reply by FullBleed** [reply]
> Not sure why you're running in WSL, since there's no speed increase to be found there.

I'm actually pretty interested in this topic...

I've seen some people report a performance increase through Linux (which, I assume, would manifest when using WSL2).

Also, would there be any potential security benefit from running through WSL2?

All of the constant pulls that go into running the various WebUIs and extensions seem kind of risky to me... and I'd love to know what best practices would be.  This space is in a "run, don't walk" mentality right now and it seems like security has taken a back seat.

**Comment by mark-wd** [comment]
You can just make a bat and use:

@echo off

set COMMANDLINE_ARGS= --upgrade

call webui.bat

**Comment by brknsoul** [comment]
>You can just make a bat and use:

>WSL 2.0

Batch files can't be used in linux.

**Comment by vladmandic** [comment]
transferring to discussions.

**Comment by kohweiteck** [comment]
I assume @ShmuelRonen are asking the more specific instructions on how to upgrade the version.
@vladmandic should describe this in more details. You can copy the instructions here and update in readme.

-----START COPY-----

For Windows user, you can update of your local build/version as followed:
(1) Go to where folder of your installed SD.Next app is
(2) Type "cmd" in the address bar
(3) Type ".\webui.bat --upgrade"
(4) Wait for few mins
(5) SD.Next is updated to he latest version
